packages/roosterjs-editor-dom/lib/table/tableMetadata.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Selectors for Table Selection
+ */
+export const enum TableMetadata {
+    /**
+     * Opacity Applied when selecting a table cell
+     */
+    SELECTION_COLOR_OPACITY = 0.7,
+    /**
+     * Class applied to table cell selected
+     */
+    TABLE_CELL_SELECTED = '_tableCellSelected',
+    /**
+     * Data set applied on a selected table cell to store the original background color
+     */
+    TEMP_BACKGROUND_COLOR = 'originalBackgroundColor',
+    /**
+     * Table cell selector
+     */
+    TABLE_CELL_SELECTOR = 'td,th',
```

Comment:
I think we don't need to have a special const for this one. Just use it directly